packages/onboard-react/src/components/HardwareWalletSteps/AddressListTable.tsx

```diff
@@ -0,0 +1,119 @@
+import { FC, ReactNode, useCallback, useMemo, useState } from "react";
+import {
+  Button,
+  AddressTablePagination,
+  TableBase,
+  ColumnOptions,
+  TransactionId,
+  Align,
+} from "@sovryn/ui";
+import { Account } from "@sovryn/onboard-hw-common";
+import { utils } from "ethers";
+import styles from "./AddressListTable.module.css";
+
+const PER_PAGE = 5;
```

Comment:
I'd consider renaming it to `DEFAULT_PER_PAGE` and having `perPage` as a prop (optional with `DEFAULT_PER_PAGE` as its default value) so it's customizable.